src/components/avatar-replay.js

```diff
@@ -0,0 +1,36 @@
+import botRecording from "../assets/avatars/bot-recording.json";
```

Comment:
might be worth renaming `bot-recording.json` and `bot-recording.mp3` to describe their contents, in case we want to have more than one kind of bot